src/Controls/samples/Controls.Sample/Pages/CustomNavigationPage.xaml.cs

```diff
@@ -7,29 +7,14 @@ namespace Maui.Controls.Sample.Pages
 	public partial class CustomNavigationPage : NavigationPage
 	{
 		public CustomNavigationPage(IServiceProvider services, MainViewModel viewModel) :
-			base(new MainPage(services, viewModel))
+			base(new MainPage(services, viewModel) { Title = "cats" })
+		//base(new TabbedPage() { Children = 
+		//	{ 
+		//		new MainPage(services, viewModel) { Title = "cats" },
+		//		new ContentPage() { Title = "empty" }
+		//	} })
 		{
 			InitializeComponent();
 		}
-
-		protected override void OnHandlerChanged()
-		{
-			base.OnHandlerChanged();
-
-#if __ANDROID__
-
-			if (Handler?.NativeView is Android.Views.View layout)
```

Comment:
Removing this for now. The AppBar behaves oddly if you navigate to a new page and it was hidden on the previous page.